### PR TITLE
Add note about Heroku service

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ ReviewMonitor is an iOS app to manage your iTunes Connect account on the go. The
 
 This project aims to make an iOS app which is more powerful than the Apple's official iTunes Connect app. Some features like multiple account login are already implemented.
 
+# How does it work?
+
+The iOS app communicates with a server hosted on Heroku, which then interacts with the Apple API. This happens using [fastlane](https://fastlane.tools), which is written in Ruby.
```
Comment (RishabhTayal) [inline README.md]: Thanks for adding this. Do you think we should also add a note about how credentials are stored and used in the app and on the server?

Comment (KrauseFx) [reply README.md]: Sure, feel free to append, you should have push access to my branch

Comment (KrauseFx) [reply README.md]: However you have to be careful that's all actually the case, that the credentials are not showing up in any logs, caches, etc.

Comment (RishabhTayal) [reply README.md]: Ok. I will skip explaining the credentials part for now. I will check the logs and cache to see if it shows up anywhere or not then will more detailed instructions.

**Pull request merged**